app/Models/EntryDAO.php

```diff
@@ -752,8 +750,9 @@ private function sqlListWhere($type = 'a', $id = '', $state = FreshRSS_Entry::ST
 
 		return array(array_merge($values, $searchValues),
 			'SELECT e.id FROM `' . $this->prefix . 'entry` e '
-			. ($joinFeed ? 'INNER JOIN `' . $this->prefix . 'feed` f ON e.id_feed=f.id ' : '')
-			. 'WHERE ' . $where
+			. 'INNER JOIN `' . $this->prefix . 'feed` f ON e.id_feed = f.id '
+			. 'WHERE f.priority >= ' . FreshRSS_Feed::PRIORITY_NORMAL . ' '
```

Comment:
* We already have a priority filter for the `'a'` case. I suggest to add an explicit priority for the other cases as well, instead of having it there.
* Maybe we should already make the priority a parameter of this `sqlListWhere()` function. I expect we will soon have cases when we want to search in all feeds, also muted/archived.

Reply:
I've done the first item. I think we could do the second one later when the question arise.